### PR TITLE
Added JavaDoc for nova.core.block.Block

### PR DESCRIPTION
**Pull request by skyem123**
I hope I didn't miss anything / do anything wrong.

### CODE REVIEW

File: src/main/java/nova/core/block/Block.java [hunk 1]
```diff
@@ -25,14 +25,30 @@
 	private final BlockAccess blockAccess = BlockAccessDummy.INSTANCE;
 	private final Vector3i position = Vector3i.ZERO;
 
+	/**
+	 * Called to get the BlockFactory that refers to this Block class.
+	 *
+	 * @return the {@link BlockFactory} that refers to this Block class.
```
Comment (RX14) [inline src/main/java/nova/core/block/Block.java]: `{@link nova.core.block.BlockFactory BlockFactory}`

File: src/main/java/nova/core/block/Block.java [hunk 2]
```diff
@@ -25,14 +25,30 @@
...
+	 */
 	public final BlockFactory getFactory() {
 		return Game.instance.get().blockManager.getBlockFactory(this.getID()).get();
 	}
 
+	/**
+	 * Get the BlockAccess that refers to this block.
+	 *
+	 * @return the {@link BlockAccess} that refers to this block.
```
Comment (RX14) [inline src/main/java/nova/core/block/Block.java]: same

**Comment by RX14** [comment]
Otherwise great

**Comment by skyem123** [comment]
Fixed the (small) issues.

**Pull request merged**